include/tables/entrance_table.h

```diff
@@ -0,0 +1,1932 @@
+
+/* 0x2D2 */ DEFINE_ENTRANCE(ENTR_SPOT12_9, SCENE_SPOT12, 9, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2D3 */ DEFINE_ENTRANCE(ENTR_SPOT12_9_1, SCENE_SPOT12, 9, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+/* 0x2D4 */ DEFINE_ENTRANCE(ENTR_SPOT12_9_2, SCENE_SPOT12, 9, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2D5 */ DEFINE_ENTRANCE(ENTR_SPOT12_9_3, SCENE_SPOT12, 9, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+
+/* 0x2D6 */ DEFINE_ENTRANCE(ENTR_SPOT12_10, SCENE_SPOT12, 10, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2D7 */ DEFINE_ENTRANCE(ENTR_SPOT12_10_1, SCENE_SPOT12, 10, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+/* 0x2D8 */ DEFINE_ENTRANCE(ENTR_SPOT12_10_2, SCENE_SPOT12, 10, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2D9 */ DEFINE_ENTRANCE(ENTR_SPOT12_10_3, SCENE_SPOT12, 10, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+
+/* 0x2DA */ DEFINE_ENTRANCE(ENTR_SPOT12_11, SCENE_SPOT12, 11, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2DB */ DEFINE_ENTRANCE(ENTR_SPOT12_11_1, SCENE_SPOT12, 11, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+/* 0x2DC */ DEFINE_ENTRANCE(ENTR_SPOT12_11_2, SCENE_SPOT12, 11, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2DD */ DEFINE_ENTRANCE(ENTR_SPOT12_11_3, SCENE_SPOT12, 11, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+
+/* 0x2DE */ DEFINE_ENTRANCE(ENTR_SPOT12_12, SCENE_SPOT12, 12, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2DF */ DEFINE_ENTRANCE(ENTR_SPOT12_12_1, SCENE_SPOT12, 12, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+/* 0x2E0 */ DEFINE_ENTRANCE(ENTR_SPOT12_12_2, SCENE_SPOT12, 12, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2E1 */ DEFINE_ENTRANCE(ENTR_SPOT12_12_3, SCENE_SPOT12, 12, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+
+/* 0x2E2 */ DEFINE_ENTRANCE(ENTR_SPOT20_2, SCENE_SPOT20, 2, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2E3 */ DEFINE_ENTRANCE(ENTR_SPOT20_2_1, SCENE_SPOT20, 2, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+/* 0x2E4 */ DEFINE_ENTRANCE(ENTR_SPOT20_2_2, SCENE_SPOT20, 2, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2E5 */ DEFINE_ENTRANCE(ENTR_SPOT20_2_3, SCENE_SPOT20, 2, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+
+/* 0x2E6 */ DEFINE_ENTRANCE(ENTR_SPOT20_3, SCENE_SPOT20, 3, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2E7 */ DEFINE_ENTRANCE(ENTR_SPOT20_3_1, SCENE_SPOT20, 3, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+/* 0x2E8 */ DEFINE_ENTRANCE(ENTR_SPOT20_3_2, SCENE_SPOT20, 3, false, true, TRANS_TYPE_FADE_WHITE, TRANS_TYPE_FADE_WHITE)
+/* 0x2E9 */ DEFINE_ENTRANCE(ENTR_SPOT20_3_3, SCENE_SPOT20, 3, false, true, TRANS_TYPE_FADE_BLACK, TRANS_TYPE_FADE_BLACK)
+
+// Labeled as "Test Area" in Map Select
```

Comment:
Is this comment all that relevant? This table is so big probably no one is going to read it, unless we really plan to have comments on every entrance group?

Reply:
this block is really special. Everything in this group is entirely unrelated, other than the fact that you can access them via map select by "test area" and changing the cutscene index

People will see it if they see test area in map select as pointing to the archery house for some reason, and care to go and check out the group of entrances that its near

Also note that I was forced to add `TEST_` to the name, cause otherwise it conflicts with the actual archery shop spawn 0 enum. So its a weird special case